en/Linking notes and files/Embedding files.md

```diff
@@ -68,3 +68,20 @@ You can also open a specific page in the PDF, by adding `#page=N` to the link de
 ```md
 ![[Document.pdf#page=3]]
 ```
+## Embed a list in a note
+
+To embed a list present in your `source_list.md` file, add the block id to your list:
```

Comment:
Thanks a lot for your review and changes. One small thing here: [[Internal links#Link to a block in a note|block identifier]] instead of pipe `|` symbol, shouldn't we provide the caret symbol `^`.

Reply:
I didn't intend to add a block identifier here, this is an alias for the link to the docs.